client/payments/data/resolvers.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import { apiFetch } from '@wordpress/data-controls';
+
+/**
+ * Internal dependencies
+ */
+import {
+	receivePaymentGateways,
+	receivePaymentGateway,
+	handleFetchError,
+} from './actions';
+
+import { API_NAMESPACE } from './constants';
+
+export function* listPaymentGateways() {
```

Comment:
I think a better name here would just be to follow the convention of the other function or to call these functions:

`fetchPaymentGateway` and `fetchPaymentGateways`.

Remember that your store should ideally match the API of our other stores since they're consumed as a package.

Reply:
Since we're following the convention, don't you think we should use 'get' in this case as well? I was thinking about changing it to `getPaymentGateways()`

Reply:
@moon0326 The reason I suggest fetch for these is they're async and the `fetch` prompts that to the API user. I tend to use words like `get` to indicate something is synchronous. *but* now I'm looking at the other stores they use `get` so lets go with that :)